src/default_tests.jsonc

```diff
@@ -15,28 +15,37 @@
     // models in a single invocation. Comment out or delete the first entry and
     // uncomment the second to run NOAA-GFDL-AM4 only for the MJO_prop_amp POD,
     // and likewise for the SM_ET_coupling POD.
+    
     {
-      "CASENAME" : "QBOi.EXP1.AMIP.001",
-      "model" : "CESM",
+      "CASENAME" : "CESM2_omip1_r1i1p1f1_gn",
```

Comment:
Remove the modifications you made to the default_tests.jsonc file. We can add a new case for the POD once the PR is merged, and we have the input files uploaded to the FTP server.